### DISCUSSION TITLE
Control the opacity of the applications window thorough keyboard shortcut

### DISCUSSION THREAD

**Discussion by Shuna322** [aunetx/blur-my-shell#494] [Show and tell]
It's not really an issue or feature request, but rather I'm sharing a bit of my knowledge and enchantment that I made.

So I really enjoy the Application blur feature with the Opacity, but during screen demonstrations or recordings I need to keep my windows crisp since video compression would ruin the quality and the text will become unreadable. Doing this manually is a bit of a hassle so I made this script and binded it to the keyboard shortcut to make my life easier.

Hopefully, this can be useful to someone, or maybe it can be integrated into the extension.

Here is the script:
https://gist.github.com/Shuna322/6c5d2af253cce5ecf1b187ec0ff6b579

And a small demo:

https://github.com/aunetx/blur-my-shell/assets/32617699/a5a96ad1-71d2-4876-83ed-fc21a15edc51

**Comment by aunetx** [comment]
Thanks a lot! I move this in discussions, but you may as well post it in reddit to improve the discoverability I guess :)
